aws/resource_aws_elasticache_replication_group.go

```diff
@@ -165,6 +187,18 @@ func resourceAwsElasticacheReplicationGroupCreate(d *schema.ResourceData, meta i
 		params.SnapshotName = aws.String(v.(string))
 	}
 
+	if _, ok := d.GetOk("transit_encryption"); ok {
```

Comment:
It will sound like a nitpick, but would you mind changing the field name to `transit_encryption_enabled`. It will better communicate the fact that this is a boolean field and make space for potential future `transit_encryption` block which may define some further settings of the encryption.

Reply:
Sure thing!